app/models/concerns/searchable_model.rb

```diff
@@ -43,6 +43,8 @@ module SearchableModel
             (attrs.map.with_index do |a, i|
               if %w(repository_rows.id repository_number_values.data).include?(a)
                 "((#{a})::text) #{like} :t#{i} OR "
+              elsif a == "('EX' || id)"
```

Comment:
why it is needed here?

Reply:
So it wouldn't run the trim_html_tags to this thing, as it is unnecessary. But I guess if I use the helper it will do it anyway. Should we run the trim function on this also?